app/services/bulk_upload/lettings/year2022/row_parser.rb

```diff
@@ -793,8 +793,10 @@ def validate_incomplete_soft_validations
       next if question.completed?(log)
 
       question.page.interruption_screen_question_ids.each do |interruption_screen_question_id|
+        next if log.form.questions.none? { |q| q.id == interruption_screen_question_id && q.page.routed_to?(log, nil) }
+
         field_mapping_for_errors[interruption_screen_question_id.to_sym]&.each do |field|
-          unless errors.any? { |e| e.options[:category] == :soft_validation && field_mapping_for_errors[interruption_screen_question_id.to_sym].include?(e.attribute) }
+          if errors.none? { |e| e.options[:category] == :soft_validation && field_mapping_for_errors[interruption_screen_question_id.to_sym].include?(e.attribute) }
```

Comment:
I didn't think I would like 'unless' when I came to ruby, however 'unless there's any errors' makes more sense to me than 'if there are none errors' which is how I parse this, totally subjective, either way works :)